sheets-to-md/output/gpu-nodes.md

```diff
@@ -1,15 +1,19 @@
+| gypsum-gpu[182-189]  | Supermicro SYS-4029GP-TRT2               | 2x Intel Xeon Silver 4116 (24 Cores 48 Threads)     | 8x NVIDIA Quadro RTX 8000 (48GB VRAM)     | 384 GiB | <ul><li>gypsum-rtx8000</li><li>gpu-preempt</li></ul>        | <ul><li>linux-ubuntu20.04-skylake_avx512</li></ul>                                                                     |
+| gypsum-gpu[190-192]  | Supermicro SYS-4029GP-TRT2               | 2x Intel Xeon Silver 4116 (24 Cores 48 Threads)     | 8x NVIDIA GeForce GTX 2080 Ti (11GB VRAM) | 384 GiB | <ul><li>gypsum-2080ti</li><li>gpu-preempt</li></ul>         | <ul><li>linux-ubuntu20.04-skylake_avx512</li></ul>                                                                     |
+| uri-gpu[001-004]     | GIGABYTE H262                            | 2x Intel Xeon(R) Platinum 8352Y (64 Cores)          | 4x NVIDIA Tesla A100 (80GB VRAM)          | 512 GiB | <ul><li>gpu</li><li>gpu-long</li></ul>                      | <ul><li>avx512</li><li>intel</li><li>linux-ubuntu20.04-icelake</li><li>a100</li></ul>                                  |
+| uri-gpu[005-008]     | GIGABYTE H262                            | 2x Intel Xeon(R) Platinum 8352Y (64 Cores)          | 4x NVIDIA Tesla A100 (80GB VRAM)          | 512 GiB | <ul><li>gpu-preempt</li><li>uri-gpu</li></ul>               | <ul><li>avx512</li><li>intel</li><li>linux-ubuntu20.04-icelake</li><li>a100</li></ul>                                  |
+| umd-cscdr-gpu001     | GIGABYTE H263                            | 2x Intel Xeon(R) Platinum 8352Y (64 Cores)          | 4x NVIDIA Tesla A100 (80GB VRAM)          | 513 GiB | <ul><li>gpu-preempt</li><li>umd-cscdr-gpu</li></ul>         | <ul><li>avx512</li><li>intel</li><li>linux-ubuntu20.04-icelake</li><li>a100</li></ul>                                  |
+| power9-gpu[001-006]  | IBM PowerSystem AC922 (PowerNV 8335-GTH) | 2x Power9 (32 Cores 128 Threads)                    | 2x NVIDIA Tesla V100 (32GB VRAM)          | 256GiB  | <ul><li>power9-gpu</li><li>power9-gpu-preempt</li></ul>     | <ul><li>power9le</li><li>ppc64le</li><li>altivec</li></li>v100</li></ul>                                               |
+| power9-gpu[007-008]  | IBM PowerSystem AC922 (PowerNV 8335-GTH) | 2x Power9 (32 Cores 128 Threads)                    | 2x NVIDIA Tesla V100 (32GB VRAM)          | 256GiB  | <ul><li>power9-gpu-osg</li><li>power9-gpu-preempt</li></ul> | <ul><li>power9le</li><li>ppc64le</li><li>altivec</li></li>v100</li></ul>                                               |
+| power9-gpu[009-016]  | IBM PowerSystem AC922 (PowerNV 8335-GTH) | 2x Power9 (40 Cores 160 Threads)                    | 4x NVIDIA Tesla V100 (16GB VRAM)          | 512GiB  | <ul><li>power9-gpu</li><li>power9-gpu-preempt</li></ul>     | <ul><li>power9le</li><li>ppc64le</li><li>altivec</li></li>v100</li></ul>                                               |
```

Comment:
```suggestion
| power9-gpu[001-006]  | IBM PowerSystem AC922 (PowerNV 8335-GTH) | 2x Power9 (32 Cores 128 Threads)                    | 2x NVIDIA Tesla V100 (32GB VRAM)          | 256GiB  | <ul><li>power9-gpu</li><li>power9-gpu-preempt</li></ul>     | <ul><li>power9le</li><li>ppc64le</li><li>altivec</li><li>v100</li></ul>                                               |
| power9-gpu[007-008]  | IBM PowerSystem AC922 (PowerNV 8335-GTH) | 2x Power9 (32 Cores 128 Threads)                    | 2x NVIDIA Tesla V100 (32GB VRAM)          | 256GiB  | <ul><li>power9-gpu-osg</li><li>power9-gpu-preempt</li></ul> | <ul><li>power9le</li><li>ppc64le</li><li>altivec</li><li>v100</li></ul>                                               |
| power9-gpu[009-016]  | IBM PowerSystem AC922 (PowerNV 8335-GTH) | 2x Power9 (40 Cores 160 Threads)                    | 4x NVIDIA Tesla V100 (16GB VRAM)          | 512GiB  | <ul><li>power9-gpu</li><li>power9-gpu-preempt</li></ul>     | <ul><li>power9le</li><li>ppc64le</li><li>altivec</li><li>v100</li></ul>                                               |
```
`v100` was surrounded by `</li>` on both sides so it wasn't formatting the list correctly.

Reply:
Ack! Thank you!

Reply:
@wmjosiah not sure if this was autogenerated? May be an issue with the script?

Reply:
No just my mistake. Thanks so much. I missed in output too :)